week3/src/components/main/FirstStage.jsx

```diff
@@ -0,0 +1,68 @@
+import React from "react";
+import "./index.css";
+const FirstStage = ({ setRecommendationType, setFirstChoice, firstChoice }) => {
+  return (
+    <>
+      <button
+        onClick={() => {
+          setFirstChoice("korea");
+        }}
+        className={firstChoice === "korea" ? "selected" : ""}
+      >
+        한식
+      </button>
+      <button
+        onClick={() => {
+          setFirstChoice("japan");
+        }}
+        className={firstChoice === "japan" ? "selected" : ""}
+      >
+        일식
+      </button>
+      <button
+        onClick={() => {
+          setFirstChoice("china");
+        }}
+        className={firstChoice === "china" ? "selected" : ""}
+      >
+        중식
+      </button>
+
+      {firstChoice === "" ? (
+        <button
+          onClick={() => {
+            setRecommendationType(2);
+          }}
+          disabled
+        >
+          NextStage
+        </button>
+      ) : (
+        <button
+          onClick={() => {
+            setRecommendationType(2);
+          }}
+        >
+          NextStage
+        </button>
+      )}
```

Comment:
```suggestion
<button
  onClick={() => setRecommendationType(2)}
  disabled={firstChoice === ""}
>
  NextStage
</button>
```
이렇게 더 간단하게 할 수 도 있겠다!